grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -739,17 +739,23 @@ class OrderController {
             redirect(action: "list")
         } else {
             def date = new Date()
-            response.setHeader("Content-disposition", "attachment; filename='\"${orderInstance.orderNumber}-${date.format("MM-dd-yyyy")}.csv\"")
+            response.setHeader("Content-disposition", "attachment; filename=${orderInstance.orderNumber}-${date.format("MM-dd-yyyy")}.csv")
```

Comment:
You can get rid of that first single quote, but I think the escaped double quote is there for a reason. I believe we had some issues with Firefox.

Reply:
Ok, will do